docs/docs/faq.md

```diff
@@ -3,3 +3,8 @@ title: FAQ
 ---
 ## Why am I still seeing vulnerable images?
 Eraser currently targets **non-running** images, so any vulnerable images that are currently running will not be removed. In addition, the default vulnerability scanning with Trivy removes images with `CRITICAL` vulnerabilities. Any images with lower vulnerabilities will not be removed. This can be configured using the [configmap](https://azure.github.io/eraser/docs/customization#scanner-options).
+
+## How is Eraser different from Kubernetes garbage collection?
+The native garbage collection in Kubernetes works a bit differently than Eraser. By default, garbage collection begins when disk usage reaches 85%, and stops when it gets down to 80%. There are a couple core benefits to using Eraser for image cleanup:
```

Comment:
can we add a link to this https://kubernetes.io/docs/concepts/architecture/garbage-collection/